src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py

```diff
@@ -17,7 +17,7 @@
 '''
 
 
-def get_github_access_token(cmd, scope_list=None):
+def get_github_access_token(cmd, scope_list=None):  # pylint: disable=unused-argument
```

Comment:
How about remove `cmd` parameter directly? As far as I know, custom function is allowed to not contain `cmd` according to https://github.com/Azure/azure-cli/blob/32d269125bec1b051cc35709ea9a631ac8686728/src/azure-cli-core/azure/cli/core/__init__.py#L807-L824
L813 will process smartly

Reply:
This `default_command_handler` is not used anywhere. Leave it to the author @calvinsID to further decide.